operators/src/processing/reprojection.rs

```diff
@@ -1276,4 +1294,43 @@ mod tests {
             ]
         );
     }
+
+    #[test]
+    fn it_derives_raster_result_descriptor() {
+        let in_proj = SpatialReference::epsg_4326();
+        let out_proj = SpatialReference::from_str("EPSG:3857").unwrap();
+
+        let in_desc = RasterResultDescriptor {
+            data_type: RasterDataType::U8,
+            spatial_reference: in_proj.into(),
+            measurement: Measurement::Unitless,
+            time: None,
+            bbox: Some(SpatialPartition2D::new_unchecked(
+                (-180., 90.).into(),
+                (180., -90.).into(),
+            )),
+            resolution: Some(SpatialResolution::new_unchecked(0.1, 0.1)),
+        };
+
+        let out_desc = Reprojection::derive_raster_result_descriptor(out_proj, &in_desc).unwrap();
+
+        assert_eq!(
+            out_desc,
+            RasterResultDescriptor {
+                data_type: RasterDataType::U8,
+                spatial_reference: out_proj.into(),
+                measurement: Measurement::Unitless,
+                time: None,
+                bbox: out_proj
+                    .area_of_use_projected::<SpatialPartition2D>()
+                    .unwrap()
+                    .into(),
+                // TODO: y resolution should be double the x resolution, but currently we only compute a uniform resolution
+                resolution: Some(SpatialResolution::new_unchecked(
+                    14_084.951_977_350_345,
```

Comment:
how did you verify the resolution? Compared with GDAL? Please add a comment how to do this.

Reply:
it's very similar to what gdal would do:

> $ gdalwarp -t_srs EPSG:3857 test_data/raster/modis_ndvi/MOD13A2_M_NDVI_2014-01-01.TIFF ndvi_3857.tif
> $ gdalinfo ndvi_3857.tif
> ...
> Pixel Size = (14052.950258048738760,-14057.881117788405390)

I just used what we had like you suggested, so there isn't really a way to verify anything.